### PR TITLE
Update mRo Control Zero Page

### PR DESCRIPTION
**Pull request by Hwurzburg**
fix some errors, add info requested by multiple users

### CODE REVIEW

**Review by pkocmoud — CHANGES_REQUESTED** [review]

File: common/source/docs/common-mro-control-zero-F7.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+================
+
+Pinouts, wiring diagrams, setup information, etc. is shown here:
+
+`mRo Control Zero <https://store.mrobotics.io/mRo-Control-Zero-F7-p/mro-ctrl-zero-f7.htm>`_
+
+Where to Buy
+============
+
+`mRobotics <https://store.mrobotics.io/>`_
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-mro-control-zero-F7.rst:73]: those have a tendency to change sometimes unexpectedly..may not with mRobotics...but many change them....I usually prefer something less volatile....but not adamant about it....your call

File: common/source/docs/common-mro-control-zero-F7.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- PPM sum input signal
+- 8x PWM outputs (all DShot capable)
+- 1x RSSI (PWM or voltage) input
+- 6x UART (serial ports total), two with HW flow control
+- USB console input with two endpoints, one defaults to MAVLink for GCS, one for SLCAN
+- 1x I2C
+- 1x SPI
+- 1x CAN
+- 1x JTAG (TC2030 Connector)
+- 3x Ultra low noise LDO voltage regulator
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-mro-control-zero-F7.rst:36]: change on your web page also...

File: common/source/docs/common-mro-control-zero-F7.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+- Tricolor LED
+
+Uncased Weight and Dimensions
+=============================
+- Weight: 5.3g (.19 oz)
+- Width: 20mm (.79”)
+- Length: 32mm (1.26”)
+
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-mro-control-zero-F7.rst:44]: will add...thanks

File: common/source/docs/common-mro-control-zero-F7.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+================
+
+All motor/servo outputs are Dshot and PWM capable. However, mixing Dshot and normal PWM operation for outputs is restricted into groups, ie. enabling Dshot for an output in a group requires that ALL outputs in that group be configured and used as Dshot, rather than PWM outputs. The output groups that must be the same (PWM rate or Dshot, when configured as a normal servo/motor output) are: 1-4, 5/6, and 7/8.
+
+More Information
+================
+
+Pinouts, wiring diagrams, setup information, etc. is shown here:
+
+`mRo Control Zero <https://store.mrobotics.io/mRo-Control-Zero-F7-p/mro-ctrl-zero-f7.htm>`_
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-mro-control-zero-F7.rst:68]: will do

File: common/source/docs/common-mro-control-zero-F7.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (pkocmoud) [reply common/source/docs/common-mro-control-zero-F7.rst:73]: Sounds good, we can leave it as is.

**Comment by Hwurzburg** [comment]
thanks for the review...merging

**Pull request merged**